web/src/pages/Upstream/Create.tsx

```diff
@@ -44,6 +43,15 @@ const Page: React.FC = (props) => {
           }
         }
 
+        if (newData?.checks?.active) {
+          const host = newData?.checks?.active.host;
+          const http_path = newData?.checks?.active.http_path;
+          const url = host + http_path;
+          const { active: activeData } = newData.checks;
+          delete activeData.host;
+          delete activeData.http_path;
```

Comment:
We would prefer to use a deep clone before these operations.
I see that the operation in line 37 is not quite suitable, can you help me to change it?

Reply:
I don't quite understand what you mean?

Reply:
OK, this is only a suggestion and not a necessity.